BookingsApi/BookingsApi.Infrastructure.Services/HearingDtoMapper.cs

```diff
@@ -8,19 +8,21 @@ public static class HearingDtoMapper
     {
         public static HearingDto MapToDto(Hearing hearing)
         {
-            var @case = hearing.GetCases().First(); // Does this need to be a lead case?
+            var cases = hearing.GetCases();
+            var @case = (cases.Count > 0) ? cases.First() : null; // Does this need to be a lead case?
```

Comment:
There should always be a case for any hearing. Please undo this change

Reply:
it is failing in the test

Reply:
Could you ensure the test setup reflects a hearing. This should not be allowed to happen